src_c/pgcompat.c

```diff
@@ -0,0 +1,43 @@
+#include <SDL.h>
+
+#if defined(SDL_VERSION_ATLEAST)
+#if !(SDL_VERSION_ATLEAST(2, 0, 5))
+/* this function is only defined in 2.0.5.
```

Comment:
Rewording`"this function is only defined in 2.0.5."` to something like `"These functions require SDL 2.0.5 or greater."` might be clearer.